etc/config/openclc.defaults.properties

```diff
@@ -0,0 +1,26 @@
+# Default settings for OpenCL C
+compilers=&clang
+defaultCompiler=clang
+postProcess=
+demangler=c++filt
+supportsBinary=false
+
+group.clang.compilers=openclcclang7:openclcclang8:openclcclang9:openclcclang10:openclcclang11:openclcclang12:openclcclang13:openclcclangdefault
+group.clang.options=-Xclang -finclude-default-header
```

Comment:
btw clang-13 does't need this option any more but maybe it is not easy to specialize?